flaskbb/forum/models.py

```diff
@@ -201,12 +201,14 @@ def save(self, user=None, topic=None):
         :param user: The user who has created the post
         :param topic: The topic in which the post was created
         """
+        current_app.pluggy.hook.flaskbb_event_post_save_before(post=self)
+
         # update/edit the post
         if self.id:
             db.session.add(self)
             db.session.commit()
-            current_app.pluggy.hook.flaskbb_event_after_post(post=self,
```

Comment:
In the future we should provide a deprecation path for hooks. Maybe putting the new one in, and registering a hook that calls the old one and raises a deprecation warning.

Reply:
Yep - but at the moment I don't think that anyone besides @haliphax. @haliphax the name of the hooks you implemented changed to ``flaskbb_event_post_save_after`` and ``flaskbb_event_post_save_before``.